dev/com.ibm.ws.security.wim.adapter.ldap/src/com/ibm/ws/security/wim/adapter/ldap/context/ContextManager.java

```diff
@@ -227,6 +235,8 @@
     /** Write to secondary server configuration. */
     private boolean iWriteToSecondary = false;
 
+    /** Buffer stream for catching JNDI output */
+    BEROutputStream bapos = new BEROutputStream();
```

Comment:
Doesn't seem we should need this to be class level. Just add create it when you add it to the env.